<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QMainWindow" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="QSplitter" name="splMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <widget class="QToolBar" name="tbrMain">
            <addaction name="actionPlay"/>
            <addaction name="actionPause"/>
            <addaction name="actionStop"/>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblSize">
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PatternSizeEditWidget" name="spnSize"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="PatternWidget" name="wgtPattern" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splLibrary">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="LibraryWidget" name="wgtLibrary" native="true"/>
       <widget class="ScriptEditWidget" name="wgtScriptEditor" native="true"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mnuMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionWavExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="mnuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="visible">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="qjsynth.qrc">
     <normaloff>:/gfx/play.png</normaloff>:/gfx/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qjsynth.qrc">
     <normaloff>:/gfx/pause.png</normaloff>:/gfx/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qjsynth.qrc">
     <normaloff>:/gfx/stop.png</normaloff>:/gfx/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionWavExport">
   <property name="text">
    <string>&amp;Export to WAV...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LibraryWidget</class>
   <extends>QWidget</extends>
   <header>LibraryWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScriptEditWidget</class>
   <extends>QWidget</extends>
   <header>ScriptEditWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PatternWidget</class>
   <extends>QWidget</extends>
   <header>PatternWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PatternSizeEditWidget</class>
   <extends>QSpinBox</extends>
   <header>PatternSizeEditWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qjsynth.qrc"/>
 </resources>
 <connections/>
</ui>
